### ISSUE TITLE
Are we checking the checksum of checksum address?

### ISSUE THREAD

**Issue by pirapira** [raiden-network/raiden-contracts#789]
Especially when the addresses are given as command line arguments.

**Comment by karlb** [comment]
No, we are not. We just check if they are addresses and then turn them into checksum addresses. See https://github.com/karlb/raiden-contracts/blob/60993da4705d73600d80bfd10ccddcdd293fdca3/raiden_contracts/deploy/__main__.py#L36-L45.
